src/components/Notification/Notification.stories.mdx

```diff
@@ -73,6 +74,57 @@ Inline Notification
   </Story>
 </Preview>
 
+# Banner Notification
+
+Banner Notification
+
+<Preview>
+  <Story name="Banner Notifiction" parameters={{ decorators: [withKnobs] }}>
+    <PresentComponent name="" width={1024}>
+      <BannersContainer
+        top={boolean('top', false)}
```

Comment:
Kostas kudos for your first PR in Ictinus! I would suggest creating a more 'user-friendly' approach as described in issue #58 (with top|bottom - right|left position). 
As a user of this container, I would expect to pass a string in the property that will ensure the position of the banner. 

For example:
Container's types:
```
interface Props  {
 position: "top-right" | "top-left" | "bottom-left" | "bottom-right";
}
```

Invocation:
`<BannersContainer position="top-right" />`

The position depends on a combination of two rules described in the above example. This way you can ensure that the user will never miss a position placement.

Reply:
Actually that's a great idea. A bug of the current implementation is that the user may insert weird props combinations (for example `<BannersContainer top bottom/>` so with your combinations system idea we avoid such cases